types/wordpress__jest-console/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for @wordpress/jest-console 3.0
+// Project: https://github.com/wordpress/gutenberg/tree/master/packages/jest-console/readme.md
+// Definitions by: Damien Sorel <https://github.com/mistic100>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="jest" />
+
+declare namespace jest {
+    interface Matchers<R> {
+        /**
+         * Ensure that `console.error` function was called.
+         */
+        toHaveErrored(): R;
+
+        /**
+         * Ensure that `console.error` function was called with specific arguments.
+         */
+        toHaveErroredWith(...args: any[]): R;
```

Comment:
I think the arguments are suppose to be `string`. Does this method require at least one argument?

Reply:
You can do `console.error(5)` if you want.

The code does not make any assumption on the number of arguments passed to the "With" methods https://github.com/WordPress/gutenberg/blob/master/packages/jest-console/src/matchers.js#L42  

I guess it's worth enforcing at least one argument. Toughts ?